image-index.md

```diff
@@ -43,32 +43,6 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
     This OPTIONAL property describes the platform which the image in the manifest runs on.
```

Comment:
line 28 should be changed from "with the following additional properties and restrictions" to "with the following requirements and restrictions" or similar